exporter/splunkhecexporter/tracedata_to_splunk_test.go

```diff
@@ -149,13 +149,9 @@ func makeSpan(name string, ts *pdata.TimestampUnixNano) pdata.Span {
 	spanLink.Attributes().InsertInt("foo", 1)
 	spanLink.Attributes().InsertBool("bar", false)
 	foobarContents := pdata.NewAttributeValueArray()
-	foobarContents.InitEmpty()
-	arrContents := pdata.NewAnyValueArray()
-	arrContents.Append(pdata.NewAttributeValueString("a"))
-	arrContents.Append(pdata.NewAttributeValueString("b"))
-	foobarContents.SetArrayVal(arrContents)
+	foobarContents.ArrayVal().Append(pdata.NewAttributeValueString("a"))
+	foobarContents.ArrayVal().Append(pdata.NewAttributeValueString("b"))
 	spanLink.Attributes().Insert("foobar", foobarContents)
```

Comment:
nicer I think this way :)

Reply:
I'm just used to slapping `InitEmpty()` to any object that happens to be in reach by now.

Reply:
I am working to remove that :)